src/rpc/rawtransaction.cpp

```diff
@@ -1280,7 +1280,7 @@ UniValue createrawcertificate(const UniValue& params, bool fHelp)
 
             std::string error;
             std::vector<unsigned char> cmt;
-            if (!Sidechain::AddScData(o.get_str(), cmt, MAX_CMT_SIZE_BYTES, false, error))
+            if (!Sidechain::AddScData(o.get_str(), cmt, MAX_CMT_SIZE_BYTES, Sidechain::CheckSizeMode::UPPER_LIMIT, error))
```

Comment:
MAX_CMT_SIZE_BYTES should be BitVectorCertificateFieldConfig::MAX_COMPRESSED_SIZE_BYTES

Reply:
Done